<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MeetingDialog</class>
 <widget class="QDialog" name="MeetingDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>637</width>
    <height>454</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>637</width>
    <height>454</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>637</width>
    <height>454</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MeetingDialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(59, 59, 59);
color: rgb(255, 255, 255);</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_3">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QPushButton" name="pb_icon">
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="res.qrc">
           <normaloff>:/tx/14.png</normaloff>:/tx/14.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>41</width>
           <height>41</height>
          </size>
         </property>
         <property name="flat">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="lb_name">
         <property name="font">
          <font>
           <family>宋体</family>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="text">
          <string>向日葵小兄弟</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pb_set">
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="res.qrc">
           <normaloff>:/images/sett.png</normaloff>:/images/sett.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>30</width>
           <height>30</height>
          </size>
         </property>
         <property name="flat">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QToolButton" name="pb_createMeet">
         <property name="minimumSize">
          <size>
           <width>68</width>
           <height>53</height>
          </size>
         </property>
         <property name="text">
          <string>创建会议</string>
         </property>
         <property name="icon">
          <iconset resource="res.qrc">
           <normaloff>:/images/message.png</normaloff>:/images/message.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>30</width>
           <height>30</height>
          </size>
         </property>
         <property name="toolButtonStyle">
          <enum>Qt::ToolButtonTextUnderIcon</enum>
         </property>
         <property name="autoRaise">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="pb_joinMeet">
         <property name="minimumSize">
          <size>
           <width>68</width>
           <height>53</height>
          </size>
         </property>
         <property name="text">
          <string>加入会议</string>
         </property>
         <property name="icon">
          <iconset resource="res.qrc">
           <normaloff>:/images/message.png</normaloff>:/images/message.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>30</width>
           <height>30</height>
          </size>
         </property>
         <property name="toolButtonStyle">
          <enum>Qt::ToolButtonTextUnderIcon</enum>
         </property>
         <property name="autoRaise">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="pb_addFriend">
         <property name="minimumSize">
          <size>
           <width>68</width>
           <height>53</height>
          </size>
         </property>
         <property name="text">
          <string>添加好友</string>
         </property>
         <property name="icon">
          <iconset resource="res.qrc">
           <normaloff>:/images/message.png</normaloff>:/images/message.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>30</width>
           <height>30</height>
          </size>
         </property>
         <property name="toolButtonStyle">
          <enum>Qt::ToolButtonTextUnderIcon</enum>
         </property>
         <property name="autoRaise">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QScrollArea" name="scrollArea">
       <property name="frameShape">
        <enum>QFrame::Box</enum>
       </property>
       <property name="widgetResizable">
        <bool>true</bool>
       </property>
       <widget class="QWidget" name="scrollAreaWidgetContents">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>618</width>
          <height>300</height>
         </rect>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="QWidget" name="wdg_list" native="true">
           <property name="minimumSize">
            <size>
             <width>600</width>
             <height>50</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>600</width>
             <height>50</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">background-color: rgb(170, 255, 255);
font: 12pt &quot;华文琥珀&quot;;
border-color: rgb(255, 170, 0);
color: rgb(0, 0, 0);</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="verticalSpacer">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>223</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections/>
</ui>
